src/it/java/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/BlockIndentation2SpacesTest.java

```diff
@@ -31,7 +31,7 @@ public class BlockIndentation2SpacesTest extends AbstractIndentationTestSupport
 
     @Override
     protected String getPackageLocation() {
-        return "com/google/checkstyle/test/chapter4formatting/rule4841indentation";
+        return "com/google/checkstyle/test/chapter4formatting/rule42blockindentation";
```

Comment:
corrected the directory name

Reply:
can you keep old name? and create issue to change it after such a massive update in such files